meeting-notes/2019/Q1/2019-02-11--js-core-dev-team-weekly.md

```diff
@@ -0,0 +1,131 @@
+- Blocked:
+    - `null`
+- Next:
+    - Let users know of `ipfs-api` => `ipfs-http-client` rename [#929](https://github.com/ipfs/js-ipfs-http-client/issues/929)
+        - <img width="500" alt="image" src="https://user-images.githubusercontent.com/1211152/52558871-045d3c00-2df4-11e9-890a-c14cf88949ac.png">
+    - DHT testing with `0.35.0-pre.0`
+    - Continue CID v1 base32 endeavour - [track it here #1440](https://github.com/ipfs/js-ipfs/issues/1440#issuecomment-461049347)
+
+@vasco-santos
+- Done:
+  - DHT Awesome Endeavor got ready and merged
+    - [ipfs/js-ipfs#/856](https://github.com/ipfs/js-ipfs/pull/856)
+  - Number of open connections under control and configurable
+    - [libp2p/js-libp2p-kad-dht#66](https://github.com/libp2p/js-libp2p-kad-dht/pull/66)
+    - [libp2p/js-libp2p#303](https://github.com/libp2p/js-libp2p/pull/303)
+    - [ipfs/js-ipfs#1800](https://github.com/ipfs/js-ipfs/pull/1800) (needs review)
+  - libp2p interop tests setup and first set of test for connect
+    - [libp2p/interop#4](https://github.com/libp2p/interop/pull/4)
+- Blocked: 
+- Next:
+  - libp2p interop tests
+    - have [libp2p/npm-go-libp2p-dep#1](https://github.com/libp2p/npm-go-libp2p-dep/pull/1) ready to be shipped
+    - have [libp2p/interop#4](https://github.com/libp2p/interop/pull/4) ready to be shipped
+    - add dht interop tests
+  - enable gossipsub implementation
+    - improve documentation for [libp2p/js-libp2p-pubsub](https://github.com/libp2p/js-libp2p-pubsub)
+    - review initial implementation of gossipsub by @Mikerah [ChainSafeSystems/gossipsub-js#15](https://github.com/ChainSafeSystems/gossipsub-js/pull/15)
+    - add pubsub support on [libp2p/js-libp2p-daemon-client](https://github.com/libp2p/js-libp2p-daemon-client)
+    
+@jacobheun
+- Done:
+  - Released [js-datastore-s3@0.2.1](https://github.com/ipfs/js-datastore-s3/blob/master/CHANGELOG.md#021-2019-02-07)
+    - New sample lock file and fixes for aws calling back multiple times
+  - Fix [aegir prerelease](https://github.com/ipfs/aegir/pull/322/files) docs  
+  - Added support for interface-stream-muxer to pull-mplex
+  - Added [basic benchmarks](https://github.com/libp2p/pull-mplex/tree/feat/interface/benchmarks) for pull-mplex and libp2p-mplex
+- Blocked:
+- Next:   
+  - Work on resolving the latest [ipfs/interop](https://github.com/ipfs/interop/pull/51#issuecomment-457632477) relay error on linux browsers
+  - Work with @achingbrain on a blog post for npm on ipfs and s3 datastore
+  - Finish [pull-mplex](https://github.com/libp2p/pull-mplex/pull/6) work and start the roll out to js-ipfs.
+    - Will do lots of testing before rolling this out
+
+@achingbrain
+- Done:
+  - Reviewed JS-IPLD API
+  - Trying to make npm-on-ipfs more stable
+  - Tracking down potential memory leak
+  - Merged & released unixfs-exporter https://github.com/ipfs/js-ipfs-unixfs-exporter/pull/16
+  - 
+- Blocked:
+ - N/a
+- Next:
+  - Inaugural Package Managers Weekly Sync
```

Comment:
@achingbrain will this be added to the community calendar for anyone to join or is it private right now?